packages/components/pie-card-container/src/index.ts

```diff
@@ -58,12 +63,48 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
                 target=${this.target || nothing}
                 rel=${this.rel || nothing}
                 aria-label=${this.aria?.label || nothing}
-                aria-disabled=${this.disabled ? 'true' : 'false'}>
+                aria-disabled=${this.disabled ? 'true' : 'false'}
+                style=${paddingCSS || nothing}>
                     <slot></slot>
                 </div>
             </a>`;
     }
 
+    /**
+     * Generates padding for the component based on `padding` values passed
+     * by the consumer.
+     *
+     *
+     * Example: 'a' or 'a, b'
+     * Single values i.e `'a'` applies to all sides and `'a, b'` applies to: top, right, bottom, left
+     *
+     * @private
+     */
+    private generatePaddingCSS (): string {
+        const { padding } = this;
+        let paddingCSS = '';
+
+        // Check if padding is empty (null, undefined, or an empty string)
+        if (!padding) {
+            return '';
+        }
+
+        const paddingArray = padding
+            .split(',')
+            .map((item) => item.trim())
+            .filter((value) => value !== '');
+
+        if (paddingArray.length > 0) {
+            paddingCSS += `var(--dt-spacing-${paddingArray[0]})`;
+
+            if (paddingArray.length > 1) {
```

Comment:
Do we only support lengths of 1 or 2?

Reply:
Yup only two values max. Min 1.

Reply:
Okay great, could we have a unit test for >2 values?